netneurotools/networks.py

```diff
@@ -59,10 +59,22 @@ def func_consensus(data, n_boot=1000, ci=95, seed=None):
 
     # group-average functional connectivity matrix desired instead of bootstrap
     if n_boot == 0 or n_boot is None:
-        corrs = [np.corrcoef(data[..., sub]) for sub in range(data.shape[-1])]
+        if type(data) is list:
+            corrs = [np.corrcoef(data[sub]) for sub in range(len(data))]
```

Comment:
Since `data` is just a list of subjects, we can iterate directly over it rather than having to index it (in the other case, subject is the last axis so we need to do the `range(data.shape[-1])` thing).

```suggestion
            corrs = [np.corrcoef(sub) for sub in data]
```